docs/source/en/installation.md

```diff
@@ -41,6 +41,7 @@ You should also install 🤗 Transformers because 🤗 Diffusers relies on its m
 <pt>
 ```bash
 pip install diffusers["torch"] transformers
+Note - PyTorch doesn't support Python 3.12 on Windows. Please use Python 3.11 instead.
```

Comment:
This should be placed outside of the code block and preferably before it :)

```suggestion
PyTorch only supports Python 3.8 - 3.11 on Windows.
```

Reply:
Hi @stevhliu, actually @patrickvonplaten suggested to place it inside the the Pytorch framework placeholder. Should I place this before the Pytorch block?

Reply:
Sorry, what I meant is it should be outside of the `bash` code block but inside the `<pt>` one like this:

```md
<pt>

PyTorch only supports Python 3.8 - 3.11 on Windows.

<code snippet here>

</pt>
```